api_app/analyzers_manager/file_analyzers/elf_info.py

```diff
@@ -0,0 +1,36 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import logging
+
+from elftools.common.exceptions import ELFError
+from elftools.elf.elffile import ELFFile
+from telfhash import telfhash
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+
+logger = logging.getLogger(__name__)
+
+
+class ELFInfo(FileAnalyzer):
+    def run(self):
+        results = {}
+        try:
+            with open(self.filepath, "rb") as file:
```

Comment:
please use `read_file_bytes` instead

Reply:
![image](https://user-images.githubusercontent.com/58381523/177570867-a18440e5-e123-4c2c-95dc-955c8680136f.png)

Reply:
`ELFFile(io.BytesIO(self.read_file_bytes()))` works. Should I use that?

Reply:
ok this is because ELFFile wants a file object. So nevermind, it's ok like that